Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -9,44 +9,45 @@ import Storage
 
 private struct BackForwardViewUX {
     static let RowHeight: CGFloat = 50
-    static let BackgroundColor = UIColor.Photon.Grey10A40
 }
 
-class BackForwardListViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, UIGestureRecognizerDelegate {
-    fileprivate let BackForwardListCellIdentifier = "BackForwardListViewController"
-    fileprivate var profile: Profile
-    fileprivate lazy var sites = [String: Site]()
-    fileprivate var dismissing = false
-    fileprivate var currentRow = 0
-    fileprivate var verticalConstraints: [NSLayoutConstraint] = []
+class BackForwardListViewController: UIViewController,
+                                     UITableViewDataSource,
+                                     UITableViewDelegate,
+                                     UIGestureRecognizerDelegate,
+                                     Themeable {
+    private let BackForwardListCellIdentifier = "BackForwardListViewController"
```

Comment:
nit, We could use the protocol for this on `BackForwardTableViewCell` since it's a small change